11.21.0

joegl opened the pull request:
# DO NOT DELETE

joegl committed:
11.20.2

joegl committed:
HSD8-1797: Adjust Course Tag Table labels and Add Translated Tag Column (#2054)

HSD8-1797: Adjust Course Tag Table labels and Add Translated Tag Column (#2054)

ahughes3 commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr2053-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://hs-colorful-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://hs-traditional-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://english-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://history-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://west-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/6914d50a83dcdd54ba529f5c

Visual Diffs:
- https://dashboard.tugboatqa.com/6914d50a83dcdd54ba529f5c/visualdiffs

joegl committed:
HSD8-1797: Remove machine_name column from Course Tag Form (#2055)

* HSD8-1797: Remove machine_name column from Course Tag Form (#2055).

ahughes3 commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr2053-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://hs-colorful-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://hs-traditional-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://english-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://history-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://west-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/6914d50a83dcdd54ba529f5c

Visual Diffs:
- https://dashboard.tugboatqa.com/6914d50a83dcdd54ba529f5c/visualdiffs

joegl committed:
Fix npm ci failures by updating theme lockfile for Node v24.11.1+ (#2057)

Fix npm ci failures by updating theme lockfile for Node v24.11.1+ (#2057)

github-actions[bot] committed:
Automated Dependency Updates 20251114 (#2056)

* Automated Dependency Updates 20251114 (#2056).
* Ignore user.mail.yml endofline rule for PHPCS scans.
---------
Co-authored-by: joegl <6943710+joegl@users.noreply.github.com>

cienvaras committed:
SHS-6432: Fix | Sass tests are failing from within the theme (#2047)

 SHS-6432: Fix | Sass tests are failing from within the theme (#2047)

ahughes3 commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr2053-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://hs-colorful-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://hs-traditional-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://english-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://history-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://west-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/6914d50a83dcdd54ba529f5c

Visual Diffs:
- https://dashboard.tugboatqa.com/6914d50a83dcdd54ba529f5c/visualdiffs

joegl committed:
Add release tag conditional to recent Node step additions in release workflow (#2058)

Add release tag conditional to recent Node step additions in release workflow (#2058)

ahughes3 commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr2053-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://hs-colorful-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://hs-traditional-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://english-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://history-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com
- https://west-6kdqkoes5f3gg3qrgtr7hhwmd9efnzpu.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/6914d50a83dcdd54ba529f5c

Visual Diffs:
- https://dashboard.tugboatqa.com/6914d50a83dcdd54ba529f5c/visualdiffs

joegl committed:
Merge branch 'develop' into 11.20.2-release

github-actions[bot] committed:
Automated Dependency Updates 20251128 (#2067)

Automated Dependency Updates 20251128 (#2067)

dalin- committed:
SHS-6448: Restructure course importers block on the dashboard to use importer URL's (#2060)

SHS-6448: Restructure course importers block on the dashboard to use importer URL's (#2060)

cienvaras committed:
SHS-6467: Bug | Overlapping text image slide indicator on Photo Album Element (#2062)

SHS-6467: Bug | Overlapping text image slide indicator on Photo Album Element (#2062)

mariannuar committed:
SHS-6434: Implement | Editors can get the bookmark/anchor to any component (#2059)

SHS-6434: Implement | Editors can get the bookmark/anchor to any component (#2059)

joegl committed:
HSD8-1804: Delete the sociology2025 site (#2069)

HSD8-1804: Delete the sociology2025 site (#2069)

joegl committed:
Add documentation for site decommissioning and deletion process (#2070)

Add documentation for site decommissioning and deletion process (#2070)

mariannuar committed:
SHS-6385: Implement: Remove unused Image Styles (#2036)

* SHS-6385: Implement: Remove unused Image Styles (#2036)

mariannuar committed:
SHS-6478: Remove email address and phone number from default view displays (#2065)

SHS-6478: Remove email address and phone number from default view displays (#2065)

github-actions[bot] committed:
Automated Dependency Updates 20251205 (#2071)

Automated Dependency Updates 20251205 (#2071)

ahughes3 commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr2053-hivyu978bb6r8iwwf8xhjpmmn6f4dxn4.tugboatqa.com
- https://hs-colorful-hivyu978bb6r8iwwf8xhjpmmn6f4dxn4.tugboatqa.com
- https://hs-traditional-hivyu978bb6r8iwwf8xhjpmmn6f4dxn4.tugboatqa.com
- https://english-hivyu978bb6r8iwwf8xhjpmmn6f4dxn4.tugboatqa.com
- https://history-hivyu978bb6r8iwwf8xhjpmmn6f4dxn4.tugboatqa.com
- https://west-hivyu978bb6r8iwwf8xhjpmmn6f4dxn4.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/692dc3205408d4dcd00f04c5

Visual Diffs:
- https://dashboard.tugboatqa.com/692dc3205408d4dcd00f04c5/visualdiffs
